airsonic-main/src/main/resources/applicationContext-db-legacy.xml

```diff
@@ -5,7 +5,7 @@
        profile="legacy">
 
     <bean id="dataSource"
-          class="org.springframework.jdbc.datasource.DriverManagerDataSource">
+          class="org.apache.commons.dbcp2.BasicDataSource">
         <property name="driverClassName" value="org.hsqldb.jdbcDriver" />
         <property name="url"
                   value="#{T(org.airsonic.player.service.SettingsService).defaultJDBCUrl}" />
```

Comment:
👍  We're getting closer to removing this separate LegacyDaoHelper class :)

Once this is in a connection pool, then there's no point to keep a separate class

Reply:
...as soon as we figure out a way to actually run `SHUTDOWN` when Airsonic closes.